Example/Example/Purchase/PurchaseLogicController.swift

```diff
@@ -118,11 +123,97 @@ final class PurchaseLogicController {
     }
   }
 
+  func payWithCashApp() {
+    if cashRequest != nil {
+      paykit?.authorizeCustomerRequest(cashRequest!)
+    }
+  }
+
+  private var cashButton: CashAppPayButton?
+
+  private var cashRequest: CustomerRequest?
+
+  static var cashData: CashAppSigningData?
+
+  private lazy var paykit: PayKit? = {
+    guard let clientId = Afterpay.cashAppClientId else {
+      assertionFailure("Couldn't get cash app client id")
+      return nil
+    }
+
+    let paykitSdkEnv = Afterpay.environment == .production ? PayKit.Endpoint.production : PayKit.Endpoint.sandbox
+    let sdk = PayKit(clientID: clientId, endpoint: paykitSdkEnv)
+    sdk.addObserver(self)
+
+    return sdk
+  }()
```

Comment:
For our demo we only use `sandbox` but maybe you want both? 
Unclear to me when you would want to give our production credentials in this context.

https://cs.github.com/cashapp/cash-app-pay-ios-sdk/blob/cad34a68717756c34505452a407d3e27c770ab7e/Demo/PayKitDemo/PayKitViewController.swift#L49